docs/theorist/bms/how_it_works.md

```diff
@@ -38,19 +38,19 @@ $\mathscr{L}(f_i)\approx\dfrac{B(f_i)}{2} - \log[P(f_i)]$
 
 where $B(f_i)=k\log[n] - 2\log[P(D|\theta^*,f_i)]$
 
-In this formulation, the goodness of fit $p(D|\theta^*,f_i)$ and likelihood $p(f_i)$ of an equation are equally and logarithmically weighted to each other - e.g., improving the fit by a factor of 2 is offset by having half the likelihood.
+In this formulation, the goodness of fit $p(D|\theta^*,f_i)$ and likelihood $p(f_i)$ of an equation are equally and logarithmically weighted to each other (e.g., improving the fit by a factor of 2 is offset by halving the likelihood).
 
-To better understand the problem, equations are framed in terms of acyclic graphs (i.e., trees)
+To better frame the problem, equations are cast as acyclic graphs (i.e., trees).
```

Comment:
I would probably rephrase this as something like "equations are modeled as". Casting is a specific term using by coders. Using it informally here might be confusing.